<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="currentHours_LCDNumber">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="currentSeconds_LCDNumber">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>70</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="currentMinutes_LCDNumber">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>50</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>20</y>
      <width>350</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Good Morning, Andrew!</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="nyHours_LCDNumber_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>130</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="nyMinutes_LCDNumber_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>130</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="nySeconds_LCDNumber_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>140</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="tokyoHours_LCDNumber_3">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>130</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="tokyoSeconds_LCDNumber_3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>140</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="tokyoMinutes_LCDNumber_3">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>130</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="londonHours_LCDNumber_4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>130</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="londonSeconds_LCDNumber_4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>140</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="londonMinutes_LCDNumber_4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>London</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>110</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>New York</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>110</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Tokyo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Current Time:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="imgDL_pushButton">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>40</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="zipCode_lineEdit">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>40</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="zipCode_label">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>40</y>
      <width>85</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Zip Code:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>301</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Maps</string>
    </property>
    <widget class="QLabel" name="imgDisplay_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>281</width>
       <height>241</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
